composer.lock

```diff
@@ -1429,16 +1429,16 @@
         },
         {
             "name": "composer/semver",
-            "version": "3.4.0",
+            "version": "3.4.2",
```

Comment:
I see potentially dangerous updates in the .lock file. Dont use `composer update` almost ever. If we do composer update we would have to go through extensive QA since almost 100% something would break :D 

If you need to update a package use the following instead:

`composer update {pakcage}[:{version}]`

If you just need to update the lock file to reflect a change you did in composer.json you can use

`composer update --lock`